.changeset/route-flag-dts-export.md

```
@@ -0,0 +1,20 @@
+
+Export `BareRouteFlag` and `ValuedRouteFlag` so a consumer can export a flag (#641).
+
+`RouteFlag<V>` is a conditional over those two interfaces, and only the alias was exported. A consumer writing the documented shape:
+
+```ts
+export const Public = defineRouteFlag('auth.public')
+```
+
+failed to emit declarations:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)